src/target/stm32h7.c

```diff
@@ -257,9 +257,20 @@ void stm32h7_configure_wdts(target_s *const target)
 bool stm32h7_probe(target_s *target)
 {
 	const adiv5_access_port_s *const ap = cortex_ap(target);
+	DEBUG_TARGET("%s: AP partno 0x%04x, part_id 0x%04x\n", __func__, ap->partno, target->part_id);
```

Comment:
This and the matching DEBUG_TARGET in stm32mp15.c should be removed as it duplicates the one in cortexm.c

Reply:
Dropped intermediate commit. I could've reverted it and squashed into cortexm, but then it becomes less pretty.